### ISSUE TITLE
Craftingstore Suppot

### ISSUE THREAD

**Issue by KermisReiziger** [plan-player-analytics/Plan#1917]
The plugin is kind of like buycraftx but has more features. So i would love to see the plugin supported :)

Website Link: https://craftingstore.net/
Spigot Link: https://www.spigotmc.org/resources/craftingstore-net-free-minecraft-donation-system-1-16-compatible.31331/

**Comment by AuroraLS3** [comment]
Their API documentation still appears to be very unfinished https://docs.api.craftingstore.net/

**Comment by Tactryl3** [comment]
Sorry about the duplicate! Didn't see this one when I clicked to see if it had been suggested. Thanks for your amazing work on this plugin!

**Comment by Tim-NL** [comment]
> Their API documentation still appears to be very unfinished https://docs.api.craftingstore.net/

Hi! The public API currently only has a handful of features. What endpoints would you require besides the general /payments endpoint? 

If I understand the Tebex extension correctly, it seems to just fetch their /payments endpoint and use the local SQL database for fetching player specific information. That should all be possible using our API.

-Tim

**Comment by AuroraLS3** [comment]
Yep the payments endpoint seems to be what is needed and that should enough to page the payment data to a local database.
The docs look great - I can't remember what they were like last year when I wrote that comment :)
